src/dataSoqlQueryTypes.ts

```diff
@@ -30,6 +30,8 @@ export interface Field {
  */
 export type SoqlQueryResult = {
   query: string;
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore //TODO
```

Comment:
Just curious, did you have to ts-ignore this because your IDE doesn't want a space between `//` and `TODO`?